carina-api/src/main/java/com/qaprosoft/carina/core/foundation/api/AbstractApiMethodV2.java

```diff
@@ -47,22 +47,22 @@
      * and @ResponseTemplatePath if present
      */
     public AbstractApiMethodV2() {
-        super("application/json");
+        super();
         setHeaders("Accept=*/*");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Accept=*/*" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ci.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1192)